src/gt4py/next/program_processors/runners/dace_fieldview/gtir_builtin_translators.py

```diff
@@ -219,11 +219,20 @@ def translate_as_fieldop(
     fun_node = node.fun
     assert len(fun_node.args) == 2
     stencil_expr, domain_expr = fun_node.args
+
+    if cpm.is_ref_to(stencil_expr, "deref"):
+        # Special usage of 'deref' as argument to fieldop expression, to pass a scalar
+        # value to 'as_fieldop' function. It results in broadcasting the scalar value
+        # over the field domain.
+        return translate_broadcast_scalar(node, sdfg, state, sdfg_builder, reduce_identity)
+
```

Comment:
This is completely optional.
But moving the check to here:
```
elif isinstance(stencil_expr, gtir.Lambda):
        pass  # Handled below
else:
        raise NotImmplemtedError(...)
```
It would make it more obvious what is handled below.
Alternatively you could move the default case also into its own function.

Reply:
Much nicer, agreed.